cmd/loadtest/loadtest.go

```diff
@@ -31,10 +40,18 @@ func RunLoadTestCmdF(cmd *cobra.Command, args []string) error {
 			Password:     "testPass123$",
 		}
 		ue := userentity.New(memstore.New(), ueConfig)
-		return simplecontroller.New(id, ue, status)
+		switch controllerType {
+		case loadtest.UserControllerSimple:
+			return simplecontroller.New(id, ue, status)
+		case loadtest.UserControllerSimulative:
+			return simulcontroller.New(id, ue, status)
+		default:
+			// This should never happen if we validated the config.
```

Comment:
I see a good opportunity for adding a `panic()` here :)

Reply:
Sounds good.